examples/basic/turbo.json

```diff
@@ -6,10 +6,15 @@
       "dependsOn": ["^build"],
       "outputs": [".next/**", "!.next/cache/**"]
     },
-    "lint": {},
+    "lint": {
+      "dependsOn": ["^lint"]
+    },
     "dev": {
       "cache": false,
       "persistent": true
+    },
+    "typecheck": {
+      "dependsOn": ["^build"]
```

Comment:
Why does typechecking require building dependencies first?

Reply:
It did at one point but doesn't anymore. Good eye.

I'm going to actually remove the typecheck script now because it will require `"^topo"` and I'd prefer an advanced technique like that not be in what will be the entrypoint to Turborepo/monorepos for many folks.